lib/moon/design/table.ex

```diff
@@ -182,4 +185,13 @@ defmodule Moon.Design.Table do
       "2xl" -> "text-moon-14 py-5 px-3"
     end
   end
+
+  defp stream_data(%{items: stream = %Phoenix.LiveView.LiveStream{}}) do
+    stream
+  end
+
+  defp stream_data(%{items: items, sort: sort}) when is_list(items) do
+    Enum.with_index(items |> add_index_as |> sort_items(sort))
+    |> Enum.map(fn {item, key} -> {key, item} end)
+  end
```

Comment:
Hey just a suggestion, why not write like this to be more readable?
```suggestion
  defp stream_data(%{items: items, sort: sort}) when is_list(items) do
   items 
   |> add_index_as 
   |> sort_items(sort)
   |> Enum.with_index(fn item, key -> {key, item} end)
  end
```
On `with_index` you can pass a function that already return the way you want the tuple, doing like that you save 1 iteration over the list as well.